docker/Dockerfile

```diff
@@ -1,18 +1,10 @@
-FROM hpcaitech/cuda-conda:11.3
+FROM hpcaitech/colossalai:0.1.8
 
+RUN conda install openmm=7.7.0 pdbfixer -c conda-forge -y \
+ && conda install hmmer==3.3.2 hhsuite=3.3.0 kalign2=2.04 -c bioconda -y
 
-# install dependency
-RUN yum install -y patch
-
-RUN conda install pytorch==1.10.0 torchvision torchaudio cudatoolkit=11.3 -c pytorch \
- && conda install setuptools=59.5.0 openmm=7.7.0 pdbfixer -c conda-forge \
- && conda install hmmer==3.3.2 hhsuite=3.3.0 kalign2=2.04 -c bioconda
-
-RUN pip install biopython==1.79 dm-tree==0.1.6 ml-collections==0.1.0 numpy==1.21.2 \
- PyYAML==5.4.1 requests==2.26.0 scipy==1.7.1 tqdm==4.62.2 typing-extensions==3.10.0.2 einops ray pyarrow
```

Comment:
just make sure `einops` included in colossalai container.

Reply:
seems some changes lost, there are many packages already in colossalai container, but `einops` not.